test/SIL.XForge.Identity.Tests/Controllers/Account/AccountControllerTests.cs

```diff
@@ -62,6 +65,37 @@ public async Task Login_IncorrectPassword()
             await env.Events.Received().RaiseAsync(Arg.Any<UserLoginFailureEvent>());
         }
 
+       [Test]
+        public async Task ForgotPassword_CorrectUsernameOrEmail()
```

Comment:
I would check to make sure that the user was updated correctly in the database.

Reply:
We have written new unit test to ensure ResetPasswordKey and ResetPasswordExpirationDate updated in database.